signal/signal.go

```diff
@@ -103,6 +177,7 @@ func (c *Interceptor) mainInterruptHandler() {
 
 		case <-c.quit:
 			log.Infof("Gracefully shutting down.")
+			c.Notifier.NotifyStop()
```

Comment:
The `c.quit` channel can only be closed by the `shutdown()` closure. So this call is unnecessary.

Reply:
OK, I did, just to be sure `quit` will never be closed by anything else than shutdown, right?